internal/usage/usage.go

```diff
@@ -77,11 +77,12 @@ const (
 	WhiteListsDeleteAfter           = "ISO-8601-formatted UTC date after which Atlas removes the entry from the whitelist."
 	BDUsersDeleteAfter              = "Timestamp in ISO 8601 date and time format in UTC after which Atlas deletes the user."
 	Force                           = "Don't ask for confirmation."
+	ForceFile                       = "Overwrite the destination file."
 	Email                           = "User’s email address."
 	LogOut                          = "Optional output filename, if none given will use the log name."
 	DiagnoseOut                     = "Optional output filename, if none given will use diagnose-archive.tar.gz."
-	LogStart                        = "Beginning of the period for which to retrieve logs."
-	LogEnd                          = "End of the period for which to retrieve logs."
+	LogStart                        = "Beginning of the period for which to retrieve logs in UNIX Epoch time."
```

Comment:
Suggest "As a Unix timestamp" instead of "in unix epoch time".

Reply:
We already refer to it as `UNIX Epoch` in our [docs](https://docs.mongodb.com/mongocli/stable/reference/atlas/logs-download/) so I'm keeping things consistent